src/App.tsx

```diff
@@ -90,6 +90,21 @@ function App({ isEmbedded = false }: { isEmbedded?: boolean }) {
     });
   }, [machine?.id, sendToSourceService]);
 
+  // TODO: Subject to refactor into embedActor
+  useEffect(() => {
```

Comment:
nit: Could you move this into a custom hook? That'll give it a bit more context as to what it's about.